wgpu-types/src/lib.rs

```diff
@@ -1881,8 +1881,8 @@ impl Extent3d {
         let block_width = block_width as u32;
         let block_height = block_height as u32;
 
-        let width = ((self.width + block_width - 1) / block_width) * block_width;
-        let height = ((self.height + block_height - 1) / block_height) * block_height;
+        let width = (((self.width + block_width) - 1) / block_width) * block_width;
```

Comment:
what happened here?

Reply:
... I thought I was fixing precidence but I didn't change it at all, let me give this another look